context/app/routes_main.py

```diff
@@ -1,15 +1,16 @@
 from flask import (render_template, current_app, abort,
                    session, request)
 
-from .utils import get_default_flask_data, make_blueprint
+from .utils import get_default_flask_data, make_blueprint, get_organs
 
 
 blueprint = make_blueprint(__name__)
 
 
 @blueprint.route('/')
 def index():
-    flask_data = {**get_default_flask_data()}
+    organs = get_organs()
+    flask_data = {**get_default_flask_data(), 'organs': organs}
```

Comment:
```suggestion
    flask_data = {**get_default_flask_data(), 'organ_count': len(get_organs())}
```

Or something like that. See comment below.

Reply:
Initially I went in this direction, but we need the icon for the kidney. I could hard code it?